tests/__init__.py

```diff
@@ -1 +0,0 @@
-
```

Comment:
what do we do when we want to load extra requirements from files? don't we have to have setup.py?